design/images/component-ui.puml

```diff
@@ -1,16 +1,58 @@
-@startuml
-node "Seedcase Docker Framework" {
- node "Front End" {
-  node "User Interface" {
-    [Landing page]
-    [User Login pages]
-    [Project list pages]
-    [Project information ages]
-    [Data list pages]
-    [Data information pages]
-    [Upload Data pages]
-    [Download Data pages]
-    [Data QC process Pages]
- }
+@startuml 
+' Component diagram for the Seedcase product - Backend: File Storage
+!include  https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Component.puml
+
+Container(bee, "Back End Environment")
+Container(management, "User and Data Management Layer")
+Container(auth, "Authentication")
+
+Container_Boundary(ui, "Front End User Interface") {
+  Container_Boundary(auth, "Authentication") {
+    Component(user_info, "Information on users", "Metadata on registered users of the Data Resource")
+  }
+  Container_Boundary(web, "Public web interface") {
+    Component(page_landing, "Landing page", $descr="Welcome and overview page that directs users to other pages")
+    Component(page_login, "Login page", $descr="To give some users access to specific pages in the Data Resource")
```

Comment:
Could "some users" be written more clearly? User 1, 2, 3, and/or 4? Maybe in the arrow texts?  (could also be specified in the table below this figure)